### DISCUSSION TITLE
Projects (beta): Support tagging multiple sprints

### DISCUSSION THREAD

**Discussion by remoun** [community/community#12724] [Projects and Issues]
Allow an issue or PR to be tagged in multiple sprints for a project. I thought I saw this previously, but we can't do it any more.

PS: I'm not sure this is the right category; please feel free to move it.

**Comment by Gervwyk** [comment]
This would be very helpful, issues often can carry over from one sprint to another, but from PM perspective it helps to see in which sprints an issue was included.
